### PR TITLE
Make it possible to disable SSE

### PR DESCRIPTION
**Pull request by 2403905**
## Description
Make it possible to disable SSE
See [#issue-491](https://github.com/owncloud/ocis-charts/issues/491)

## How Has This Been Tested?
locally on minikube

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

<!-- or create documentation PR in https://github.com/owncloud/docs-ocis/tree/master/modules/ROOT/pages/deployment/container>
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation generated (`make docs`) and committed
- [ ] Documentation ticket raised: <link>
- [ ] Documentation PR created: <link>

### CODE REVIEW

File: charts/ocis/values.yaml [hunk 1]
```diff
@@ -185,6 +185,9 @@ features:
       # -- Reference to a ConfigMap containing images that can be referenced from the html mail template.
       # This ConfigMap is optional and can be omitted when images are not used.
       htmlMailImagesConfigRef: "html-mail-images"
+  # -- Disables server-sent events (sse). When disabled, clients will no longer receive sse notifications.
```
Comment (d7oc) [inline charts/ocis/values.yaml:188]: A line break as separator would be good here.

**Review by d7oc — COMMENTED** [review]
LGTM aside the small layout problem

**Comment by 2403905** [comment]
@d7oc I added conditional running of the SSE service. Please have a look.

File: charts/ocis/docs/values.adoc.yaml [hunk 2]
```diff
@@ -186,6 +186,10 @@ features:
       # -- Reference to a ConfigMap containing images that can be referenced from the html mail template.
       # This ConfigMap is optional and can be omitted when images are not used.
       htmlMailImagesConfigRef: "html-mail-images"
+
+  # -- Disables server-sent events (sse). When disabled, clients will no longer receive sse notifications.
+  disableSse: false
```
Comment (wkloucek) [inline charts/ocis/docs/values.adoc.yaml:191]: I'd prefere this to be in a sse subsection like:

```
features:
  sse:
    disabled: false
```

This makes it future proof regarding more sse settings

**Review by wkloucek — CHANGES_REQUESTED** [review]

**Review by d7oc — APPROVED** [review]
LGTM

**Pull request merged**